src/Markdown.MAML/Parser/MarkdownParser.cs

```diff
@@ -569,7 +569,7 @@ public static string UnwindMarkdownCharsEscaping(string spanText)
             _PrevString = "foo";
             _InList = false;
 
-            return Regex.Replace(spanText, "([^\r\n]*)(\r\n|$)", new MatchEvaluator(LineBreaksMatchEvaluater)).Replace(" \r\n", "\r\n").Trim();
+            return spanText;
```

Comment:
That crappy regex was here to unify all line ends to `\r\n` (which we rely on for tests at least), independent of the platform used. Why did you change it?

Is it ofsetted by the `// in this case ignore` line below?